cmd/opm/alpha/bundle/generate.go

```diff
@@ -44,11 +46,18 @@ func newBundleGenerateCmd() *cobra.Command {
 	bundleGenerateCmd.Flags().StringVarP(&outputDirArgs, "output-dir", "u", "",
 		"Optional output directory for operator manifests")
 
+	bundleGenerateCmd.Flags().BoolVarP(&createCSVArgs, "create-csv", "y", false,
+		"Creates ClusterServiceVersion from a series of kubernetes manifests in a registry+v2 bundle")
```

Comment:
Well, I feel at this point, it all depends on what we are leaning towards. It certainly is a step to v2, but it might be classified as v1.  If we do not need a new type and just use the old CSV file, ie: have users to provide a CSV but allow them to provide fewer things. Then we can call this v1. If we are using the new type of file to provide non-inferable information, we should call it v2.

Reply:
We're adding metadata to the bundle, but none of that is used by clients outside of the CSV generator (e.g. OLM, operatorhub). Since there's no change to the external and storage formats from this change -- still a v1 bundle image with a CSV in it -- we've basically introduced a new config file for v1 bundles. Because of this, I think it's fair to classify your work as an extension of v1.

I just want to be careful of introducing the concept of a v2 bundle before we actually iterate the external bundle version.